tensorflow_addons/image/tests/translate_ops_test.py

```diff
@@ -74,3 +76,23 @@ def test_translate_xy():
     )
 
     np.testing.assert_equal(translated.numpy(), expected)
+
+
+@pytest.mark.usefixtures("maybe_run_functions_eagerly")
+@pytest.mark.parametrize("dtype", _DTYPES - {tf.dtypes.float16})
```

Comment:
this is why I don't like to test against other libraries' function as they might not support certain dtype.